service/gcs/runtime/runc/utils.go

```diff
@@ -61,9 +61,13 @@ func (r *runcRuntime) makeContainerDir(id string) error {
 	return nil
 }
 
+func (r *runcRuntime) getContainerStateDir(id string) string {
+	return filepath.Join(r.runcLogBasePath, id, "state")
```

Comment:
So matching the host changes I think this should now be `/run/gcs/c/<id>/bundle`. IE: runc logs get written to the runc OCI bundle dir for the container. Seems like a nice place to find them.